src/img/index.js

```diff
@@ -84,14 +84,15 @@ import scythe from './tools/scythe.png'
 import shovel from './tools/shovel.png'
 import wateringCan from './tools/watering-can.png'
 
+import coal from './ores/coal.png'
 import stone from './ores/stone.png'
-import bronzeOre from './ores/bronze-ore.png'
+import bronzeOre from './ores/stone-bronze-ore.png'
 import bronzeIngot from './ores/bronze-ingot.png'
-import ironOre from './ores/iron-ore.png'
+import ironOre from './ores/stone-iron-ore.png'
 import ironIngot from './ores/iron-ingot.png'
-import silverOre from './ores/silver-ore.png'
+import silverOre from './ores/stone-silver-ore.png'
 import silverIngot from './ores/silver-ingot.png'
-import goldOre from './ores/gold-ore.png'
+import goldOre from './ores/stone-gold-ore.png'
```

Comment:
I'm curious, what's the thinking behind the new `stone-` prefix for these?

Reply:
me being unsure about replacing the old images 😅 . but it seems like we agree they turned out well so let me update this now..